contracts/AuraBribesProcessor.sol

```diff
@@ -146,7 +146,7 @@ contract AuraBribesProcessor is CowSwapSeller {
     /// @dev
     /// Step 2.b
     /// Swap WETH -> graviAURA or WETH -> AURA
-    function swapWethForAURA(Data calldata orderData, bytes memory orderUid) external {
+    function swapWethForAuraOrBveAura(Data calldata orderData, bytes memory orderUid) external {
```

Comment:
@shuklaayush, the function name also needs to be changed on the tests then.

Reply:
Yeah I'm undoing it